### ISSUE TITLE
AttributeError: module 'flask.json' has no attribute 'JSONEncoder'

### ISSUE THREAD

**Issue by PaulisMatrix** [skozlovf/flask-json#12]
``` 
File "/opt/hostedtoolcache/Python/3.9.13/x64/lib/python3.9/site-packages/flask_json.py", line 423, in <module>
class JSONEncoderEx(json.JSONEncoder):
AttributeError: module 'flask.json' has no attribute 'JSONEncoder'
```
Github actions runner is failing to build new commits due to the above error. 
Not sure why it suddenly decided to break. Below are the flask related versions Im having:

```
Flask==2.1.0
Flask-Cors==3.0.9
Flask-JSON==0.3.5
Flask-SQLAlchemy==2.5.1
```
Possibly related issue? https://github.com/skozlovf/flask-json/issues/10
But I haven't updated flask to 2.3.x though. Don't know why the sudden breakage. 

Any help would be appreciated, thank you!!

**Comment by skozlovf** [comment]
That's strange, flask 2.1.0 should work.
Are you sure it's `Flask==2.1.0`?

PS I have `2.3` compatible code but have not released it yet.

**Comment by PaulisMatrix** [comment]
Hey @skozlovf , yeah I'm having `Flask==2.1.0` version only. 

Oh alright. Can you release it? Perhaps I can try using that one.

**Comment by PaulisMatrix** [comment]
@skozlovf , made few changes to the actions runner it worked.

`Python 3.9.13 -> 3.9.6`
`Ubuntu latest -> 20.4`

Python pkg version dependency is hard :/

**Comment by skozlovf** [comment]
@PaulisMatrix Yes, I'll try to release this week. Pretty busy these days.

**Comment by skozlovf** [comment]
Fixed in `0.4.0`.
